Autopark/Autopark.DataAccess/Specifications/FreeOnlyCarSpecification.cs

```diff
@@ -0,0 +1,23 @@
+using System.Linq.Expressions;
+using Autopark.DataAccess.Entities;
```

Comment:
you have two same specification for different entities better create generic specificarion class and when you will create instance of specification then specify your entity